APMrover2/control_modes.cpp

```diff
@@ -134,6 +134,23 @@ bool AP_RCSwitch_Rover::in_rc_failsafe() const
             (rover.failsafe.bits & FAILSAFE_EVENT_RC));
 }
 
+void AP_RCSwitch_Rover::do_aux_function_change_mode(Mode &mode,
```

Comment:
I think AP_RCSwitch_Rover::do_aux_function_change_mode should be moved to APMRover2/AP_RCSwitch.cpp.